packages/worker/src/worker.ts

```diff
@@ -272,11 +273,17 @@ function includesDeps<T extends WorkerSpec>(
 
 export function addDefaults<T extends WorkerSpec>(options: WorkerSpecOptions<T>): WorkerOptionsWithDefaults<T> {
   const { workDir, ...rest } = options;
+  // If workDir is defined, workflowsPath defaults to workflows.js if workflows does not exist
+  let workflowsPath = undefined;
+  if (workDir) {
+    const workflowsDir = resolve(workDir, 'workflows');
+    workflowsPath = existsSync(workflowsDir) ? workflowsDir : resolve(workDir, 'workflows.js');
```

Comment:
This is incorrect, a user might have both `workflows/workflowA.ts` and `workflows.ts`.
We also support reading either `js` or `ts` extensions (for better source maps, default when running with `ts-node`).

If you address all of these issues along with handling interceptors loading when `workflowsPath` is a file than we can merge this.

Reply:
Thanks. To be rigorously correct, though, don't these checks need to be performed at runtime? I don't think watch is smart enough to know that adding or removing a workflows file/folder will invalidate this logic.

I wish I knew how this path is loaded at runtime. For example, where is the code that loads index.js now?

Reply:
What do you mean by "at runtime"?
I watch mode, nodemon will restart the process anytime a file changes and this logic will be re-evaluated.
`index.{j,t}s` is loaded by webpack in `isolate-builder.ts`.

Reply:
I see. I suppose this needs to work with any custom loader, e.g., babel-loader as well as ts-loader.